src/ert/_c_wrappers/enkf/site_config.py

```diff
@@ -63,106 +123,53 @@ def from_config_dict(self, config_dict):
-                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}")
+            except (ValueError, OSError) as e:
+                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}: {e}")
 
         for job_path in config_dict.get(ConfigKeys.INSTALL_JOB_DIRECTORY, []):
             if not os.path.isdir(job_path):
                 print(f"WARNING: Unable to locate job directory {job_path}")
                 continue
             files = os.listdir(job_path)
+            print(files)
             for file_name in files:
                 full_path = os.path.join(job_path, file_name)
                 if os.path.isfile(full_path):
                     try:
                         new_job = ExtJob(
                             config_file=full_path,
                             private=False,
-                            license_root_path=__license_root_path,
+                            license_root_path=license_root_path,
                         )
                         new_job.convertToCReference(None)
                         ext_job_list.add_job(new_job.name(), new_job)
                     except (ValueError, OSError):
-                        print(f"WARNING: Unable to create job from {full_path}")
+                        print(
+                            f"WARNING: Unable to create job from {full_path}"
+                            f" in directory {job_path}"
+                        )
 
         ext_job_list.convertToCReference(None)
 
         # Create varlist
-        site_config_env_vars: EnvironmentVarlist = site_config._get_env_var_list()
-        environment_vars = config_dict.get(ConfigKeys.SETENV, [])
-
-        for elem in environment_vars:
-            site_config_env_vars.setenv(elem[ConfigKeys.NAME], elem[ConfigKeys.VALUE])
-
-        return site_config
-
-    def __repr__(self):
-        if not self._address():
-            return "<SiteConfig()>"
-        return (
-            "SiteConfig(config_dict={"
-            + f"{ConfigKeys.INSTALL_JOB}: ["
-            + ", ".join(str(job) for job in self.get_installed_jobs())
-            + "],"
-            + f"{ConfigKeys.SETENV}: {self._get_env_var_list()}"
-            + "})"
-        )
-
-    @property
-    def config_file(self):
-        return self._get_config_file()
+        env_vars = EnvironmentVarlist()
+        dict_vars = config_dict.get(ConfigKeys.SETENV, [])
 
-    def get_installed_jobs(self):
-        return self._get_installed_jobs().setParent(self)
+        for elem in dict_vars:
+            env_vars.setenv(elem[ConfigKeys.NAME], elem[ConfigKeys.VALUE])
 
-    def get_license_root_path(self) -> str:
-        return self._get_license_root_path()
+        dict_paths = config_dict.get("UPDATE_PATH", [])
 
-    @classmethod
-    def getLocation(cls) -> str:
-        return cls._get_location()
+        for elem in dict_paths:
+            env_vars.update_path(elem[ConfigKeys.NAME], elem[ConfigKeys.VALUE])
```

Comment:
this seems new - where does it come from / why do we need it?

Reply:
it was simply missing before, in the dict path, wasn't it? : )

Reply:
It was previously done as a magic incantation by the trolls inside the ERT. Now it is done here. Less fun, I know.

Reply:
I see, thank you for sharing a glimpse of the dark magic of ERT with me ^^